installer/charts/values.yaml.tpl

```diff
@@ -10,6 +10,9 @@
 {{- $ingressDomain := required "OpenShift ingress domain" .OpenShift.Ingress.Domain -}}
 {{- $ingressRouterCA := required "OpenShift RouterCA" .OpenShift.Ingress.RouterCA -}}
 {{- $minIOOperatorEnabled := or $tpa.Enabled $quay.Enabled -}}
+{{- $odfEnabled := or $tpa.Enabled $quay.Enabled -}}
+{{- $odfNamespace := "openshift-storage" -}}
```

Comment:
Maybe the namespace/channel definition could be moved closer to where they're being used.

Reply:
Sure thing, makes sense

Reply:
Since namespace and enabled is use in openshift, subscription and infrastructure leaving here. Moving channel to be under subscription section.